src/pdf/ShoppingList/ShoppingListPDF.tsx

```diff
@@ -0,0 +1,235 @@
+"use client";
+
+import React from "react";
+import { Document, Page, Text, View, Image, StyleSheet } from "@react-pdf/renderer";
+import {
+    BlockProps,
+    ClientSummary,
+    Item,
+    ShoppingListPDFProps,
+} from "@/pdf/ShoppingList/dataPreparation";
+
+const styles = StyleSheet.create({
+    paper: {
+        margin: "0.75in",
+        lineHeight: 1.5,
+    },
+    cell: {
+        display: "flex",
+        padding: "20 5 5",
+    },
+    pdfHeader: {
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "space-between",
+    },
+    logoStyling: {
+        display: "flex",
+        maxWidth: "20%",
+        alignSelf: "center",
+    },
+    infoBlock: {
+        display: "flex",
+        flexDirection: "row",
+        borderStyle: "solid",
+        border: 1,
+    },
+    infoCell: {
+        display: "flex",
+        width: "50%",
+        padding: 5,
+    },
+    table: {
+        display: "flex",
+        flexDirection: "column",
+        borderStyle: "solid",
+        border: 1,
+        padding: 5,
+    },
+    tableRow: {
+        display: "flex",
+        flexDirection: "row",
+        borderBottomStyle: "solid",
+        borderBottom: 1,
+        paddingVertical: 4,
+        textAlign: "center",
+    },
+    tableItemDescription: {
+        width: "25%",
+    },
+    tableQuantity: {
+        width: "35%",
+    },
+    tableNotes: {
+        width: "30%",
+    },
+    tableDone: {
+        width: "10%",
+    },
+    title: {
+        fontSize: "30pt",
+        fontFamily: "Helvetica-Bold",
+    },
+    subtitle: {
+        fontSize: "24pt",
+        fontFamily: "Helvetica",
+    },
+    keyText: {
+        fontSize: "12pt",
+        fontFamily: "Helvetica-Bold",
+        textTransform: "capitalize",
+    },
+    normalText: {
```

Comment:
`textTransform` and `paddingTop` are now inline styles, and `input` has been combined with `keyText`. `normalText` and `keyText` are not combined because the `fontFamily` doesn't override (tried doing a `boldText {}` and replacing `keyText` with `styles = [normalText, boldText]` but that didn't work out so decided to leave it this way

Reply:
Think you could probably set the normalText in a higher up element, so say in the page have Helvetica as default, then you shouldn't need normal text everywhere!